tests/20-SideMenu/BoardsMenuSection.test.tsx

```diff
@@ -26,62 +27,77 @@ describe("Regular", () => {
   test("Renders section with board menu items", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByText(Regular!.args!.title!)).toBeVisible;
+
+    const boards = screen.getAllByRole("link");
+    expect(boards).toHaveLength((Regular.args as BoardsSectionProps).boards!.length);
+    boards.forEach((board, i) => {
+      expect(board).toHaveTextContent(`!${(Regular.args as BoardsSectionProps).boards[i].slug}`);
+    });
   });
   
-  //Not sure if this should be it's own test or part of the previous test
   test("Board menu items link to boards", async () => {
     render(<Regular />);
-  
-    //TODO: fill this
+    const actionReturn = jest.fn();
+    mocked(action).mockReturnValue(actionReturn);
+    
+    const boards = screen.getAllByRole("link");
+    for (const board of boards) {
+      fireEvent.click(board);
+      await waitFor(() => {
+        expect(action).toBeCalledWith("#slug");
+      });
+    };
   });
   
   test("Correctly renders board without updates", async () => {
     render(<Regular />);
-
-    //Not sure .toBe is the right thing to expect here, but can't try it to figure it out because Testing Library can't find my aria labels
-    expect(screen.getByText("!oncie-den")).toBe(screen.getByRole("link", { name: "oncie-den"}));
+    
+    expect(screen.getByRole("link", { name: "oncie-den"})).toHaveTextContent("!oncie-den");
   });
   
   test("Correctly marks boards with updates", async () => {
```

Comment:
Another thing you might want to add here, is a check to ensure that the notification dot is visible. It's ok if you don't check for muted, just that it's there.

Reply:
Okay, I wasn't sure if that fell under incorrectly testing implementation details. I can add it.